projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.component.ts

```diff
@@ -17,6 +17,10 @@ import { IGX_EXPANSION_PANEL_COMPONENT, IgxExpansionPanelBase } from './expansio
 
 let NEXT_ID = 0;
 
+export interface AnimationSettings {
```

Comment:
👍

Reply:
Could event be moved to core at some point.

Reply:
Should I move it now? AFAIK the only components that expose animations of this type are the panel and the banner (using the panel), so maybe it's a bit unneeded

Reply:
No need, this one is an improvement already :)